lib/domgen/jpa/model.rb

```diff
@@ -655,13 +660,16 @@ def pre_verify
         end
 
         entity.queries.select { |query| query.jpa? && query.jpa.no_ql? }.each do |query|
+          ignore_deleted_at_str = 'IgnoringDefaultCriteria'
```

Comment:
The variable could be renamed from `ignore_deleted_at_str` to `ignore_default_criteria_str`